docs/blog/unettest.md

```diff
@@ -0,0 +1,288 @@
+# UNETTEST
+
+![What is the answer, O Deep Thought](https://docs-unettest.s3.us-east-2.amazonaws.com/o_deep_thought.png)
+
+When I write software, there are many ways to ask a question. I could go to
+Stack Overflow or a smarter coworker, but I usually like to ask my computer
+first.  What do I do if I feel like my computer is making me wait [seven and a
+half million years](https://hitchhikers.fandom.com/wiki/Deep_Thought) to get an
+answer? 
+
+I would want to revisit the question of how to ask questions. If I want to know
+the Ultimate Answer to Life, the Universe, and Everything, I must ask a damn
+Ultimate Question. Otherwise I will end up with an answer like 42 that took tons
+of effort to calculate.
+
+42 is an interesting data point and it might be useful, but in order to figure
+out what's going on, I might also like to know the answer to Death, the
+Universe, and Everything. And I might like to ask a question about just the
+Universe on its own, maybe a question or two on that Everything bit, and perhaps
+a couple of questions about a bowl of petunias. I cannot afford to wait so long
+between each answer. I have things to do.
+
+I do not have time to build the Earth in order to ask my questions so instead I
+use TDD. TDD is awesome. I can ask many questions quickly, and once I have asked
+a question, I am confident it will remain answered. If I have a question on how
+something should work, I write a test describing the behavior. Then I run my
+test and write code to make the test green. Great stuff. If I wonder about a
+corner case, I write a test case. When my code turns red, I make it better.
+
+I was writing NGINX configurations some time ago. I was new to NGINX and trying
+to figure it out. Their documentation (lovely though it is) can be impenetrable.
+I was grokking the config language by looking at existing configurations and
+mimicking what they do. I could not just hack at it though because this bouquet
+of CONF files is handling all of our network traffic--it is largely a
+look-don't-touch affair. If I had a question, I could not ask it of my computer
+easily. NGINX does not have support for unit tests (to my knowledge), or rapid
+feedback of any kind. If I want to know the answer to a question, I cannot
+write a test and run it locally. I would have to deploy that question to
+NonProd and wave around my dowsing rod to try and sense if things are both
+working and not broken. High cost, low reliability.
+
+This is silly in the best of times, but workable--no reason to fix what ain't
+broke. Until I started scratching around the corners of an NGINX edge case...
+The only way I was able to figure out what on Earth was going on was by walking
+through all the different potential paths of logic, and ugh, that was not
+pretty. Let's get into it.
+
+## The Monolith and the Microservice
```

Comment:
This section definitely has the right tone, and gets to the point, this is what I was looking for up above.